modules/ROOT/pages/6.0-release-notes-core-changes.adoc

```diff
@@ -0,0 +1,296 @@
+= TinyMCE 6.0: the core changes
+:navtitle: Core changes
+:description: The core changes for TinyMCE 6.0
+:keywords: releasenotes, bugfixes, defaults
+
+This section documents the three core changes made to {productname} 6.0.
+
+This section also includes four tables that set out the:
+
+* renamed APIs, Commands, Plugins, Options, and UI &amp; UX elements
+* values changed in a default {productname} 6.0 instance.
+* previously deprecated and now removed items.
+* never documented, and consequently never supported, items that have now been removed.
+
+
+* xref:core-changes[Core changes]
+* xref:things-we-renamed[Things we renamed]
+* xref:default-value-changes[Default value changes]
+* xref:previously-deprecated-items-now-removed[Previously deprecated items now removed]
+* xref:previously-undocumented-items-removed-without-prior-deprecation[Previously undocumented items removed without prior deprecation]
+
+
+// tag::core-changes[]
+[[core-changes]]
+== Core changes
+
+. {productname} 6.0 now requires a browser capable of running https://tc39.es/ecma262/2020/[ECMAScript 2020] (ES2020).
++
+NOTE: In particular, this means {productname} 6.0 does not support Internet Explorer 11.
++
+. No Premium Plugins being released with {productname} 6.0 are backwards compatible with earlier versions of {productname}.
+. {productname} 6.0 is now available in three further language:
+* Hindi (hi);
+* Malay (ms); and
+* Vietnamese (vi)
+
+// end::core-changes[]
+
+
+// tag::things-we-renamed[]
+[[things-we-renamed]]
+== Things we renamed
+
+[cols="1,1,1,1,1"]
+|===
+| old name                            | new name                         | element                             | notes                                                                                                                                                                                                                                                                                                         | for more information
+
+| `blockformats`                      | `blocks`                         | Menu item                           | No behavioural changes.                                                                                                                                                                                                                                                                                       |
```

Comment:
We need to use US english in the docs, so I believe this should be this instead:
```suggestion
| `blockformats`                      | `blocks`                         | Menu item                           | No behavioral changes.                                                                                                                                                                                                                                                                                       |
```